serving/docker/lmi.Dockerfile

```diff
@@ -9,31 +9,33 @@
 # or in the "LICENSE.txt" file accompanying this file. This file is distributed on an "AS IS"
 # BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, express or implied. See the License for
 # the specific language governing permissions and limitations under the License.
-ARG version=12.1.1-cudnn8-devel-ubuntu22.04
+ARG version=12.4.1-devel-ubuntu22.04
```

Comment:
Why remove cudnn8?

Reply:
should we manually install cudnn?

Reply:
Going with the same base as vllm: https://github.com/vllm-project/vllm/blob/v0.4.2/Dockerfile#L10

Reply:
Onnxruntime need cudnn

Reply:
Will add it back